src/bootstrap/test.rs

```diff
@@ -774,6 +774,19 @@ impl Step for RustdocJSNotStd {
     }
 }
 
+fn check_if_browser_ui_test_is_installed(npm: &Path, global: bool) -> bool {
```

Comment:
We always call this with global true and false; can you make a wrapper function that calls this twice (with and without global)?

Reply:
Sure!